src/types.ts

```diff
@@ -49,6 +49,8 @@ export interface InterventionProps {
   v2?: Intervention;
   criticals?: CriticalIssue;
   atf4?: ATF4Intervention;
+  // An extra field present in input for the browse schemes page only
```

Comment:
Still using `gjScheme`, and thus editing this type. The only reason is so we can reuse `InterventionLayers.svelte`. As requirements for this page evolve, maybe we revisit this too, and render the features differently on the map anyway. (Coloring by funding status, scheme, etc?)